### PR TITLE
demo-app added

### PR DESCRIPTION
**Pull request by Ganji-NaveenKumar**
External React Demo
demo-app(react-app) added Demo-app

### CODE REVIEW

File: Demo-app/demo-app/public/assets/Component.png
Comment (saicharan-zemoso) [inline Demo-app/demo-app/public/assets/Component.png:1]: check the font and download the font if not available.

File: Demo-app/demo-app/public/assets/Ethereum.svg
Comment (saicharan-zemoso) [inline Demo-app/demo-app/public/assets/Ethereum.svg:1]: place the images under public/assets/icons or images

File: Demo-app/demo-app/src/components/molecules/IconTypography/index.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import Icon from "../../atoms/Icon/index";
+import TypographyBody from "../TypographyBody";
+import styled from "@emotion/styled";
+interface IconTypographyProps {
+  iconSrc: string;
+  typographyHeader: string;
+  typographyBody: string;
+  variant?: "body1" | "caption" | "h5" | "subtitle1";
+}
+const StyledDiv = styled.div`
```
Comment (saicharan-zemoso) [inline Demo-app/demo-app/src/components/molecules/IconTypography/index.tsx]: don't use div

File: Demo-app/demo-app/src/components/molecules/ItemCard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import { Box } from "@mui/material";
+import React from "react";
+import TypographyBody from "../TypographyBody";
+import IconTypography from "../IconTypography";
+import Data from "../../../utils/constants/data";
+import styled from "@emotion/styled";
+const IconTypoStyle = styled.div`
```
Comment (saicharan-zemoso) [inline Demo-app/demo-app/src/components/molecules/ItemCard/index.tsx]: use mui component

File: Demo-app/demo-app/src/components/molecules/IconTypography/index.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import Icon from "../../atoms/Icon/index";
+import TypographyBody from "../TypographyBody";
+import styled from "@emotion/styled";
+interface IconTypographyProps {
+  iconSrc: string;
+  typographyHeader: string;
+  typographyBody: string;
```
Comment (saicharan-zemoso) [inline Demo-app/demo-app/src/components/molecules/IconTypography/index.tsx]: rename the props use better property names

File: Demo-app/demo-app/src/components/molecules/ItemCard/index.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+              iconSrc={item.image}
+              typographyHeader={item.Header}
+              typographyBody={item.Body}
+            ></IconTypography>
+          </Box>
+          <Box>
+            <StyledTypography>
+              <TypographyBody
+                Header={item.TotalAmount}
+                Body={item.Percentage}
```
Comment (saicharan-zemoso) [inline Demo-app/demo-app/src/components/molecules/ItemCard/index.tsx]: same here

**Review by saicharan-zemoso — CHANGES_REQUESTED** [review]
please provide a new screenshot of the component after resolving all the comments.

**Comment by Ganji-NaveenKumar** [comment]
image

**Review by saicharan-zemoso — APPROVED** [review]
looks good to me

**Pull request merged**